atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/StreamTest.java

```diff
@@ -150,47 +185,206 @@ private long storeAndCheckByteStreams(int size) throws IOException {
         Random rand = new Random();
         rand.nextBytes(bytesToStore);
 
-        final long id = timestampService.getFreshTimestamp();
-        PersistentStreamStore store = StreamTestStreamStore.of(txManager, StreamTestTableFactory.of());
-        txManager.runTaskWithRetry(t -> {
-                    store.storeStreams(t, ImmutableMap.of(id, new ByteArrayInputStream(bytesToStore)));
-                    store.markStreamAsUsed(t, id, reference);
-                    return null;
-                });
+        final long id = storeStream(bytesToStore, reference);
 
-        verifyLoadingStreams(id, bytesToStore, store);
+        verifyLoadingStreams(id, bytesToStore);
 
         store.storeStream(new ByteArrayInputStream(bytesToStore));
-        verifyLoadingStreams(id, bytesToStore, store);
+        verifyLoadingStreams(id, bytesToStore);
+
+        return id;
+    }
+
+    @Test
+    public void shouldQuicklyGetFirstByte() throws IOException {
+        long millisFor100mb = measureMillisToGetFirstByteFromStreamStoreWithMbs(100);
+        long millisFor1mb = measureMillisToGetFirstByteFromStreamStoreWithMbs(1);
```

Comment:
We should put these in benchmarking if anywhere. Circle seems to have random spikes every so often.